resource-aggregate/pb/commands.proto

```diff
@@ -385,3 +385,24 @@ message ConfirmResourceCreateResponse {
     AuditContext audit_context = 100;
 }
 
+message OnlineStatus {
+    bool value = 1;
```

Comment:
I would change it to enum (string). online/offline for now. We might extend it easily by connected, authorized, online ...